nbbuild/templates/projectized.xml

```diff
@@ -86,7 +86,46 @@
         <property name="have-custom-javac-task" value="true" />
     </target>
 
-    <target name="build-init" depends="basic-init,jdk-8-check,set-buildnumber,-define-custom-javac-task">
+    <target name="-init-bootclasspath-prepend" depends="basic-init,jdk-8-check,set-buildnumber,-define-custom-javac-task" unless="have-jdk-1.9">
```

Comment:
It might be better to augment the conditions below that test for `test.use.jdk.javac` to also include `have-jdk1.9`, rather than skipping this completely, so that the `bootclasspath.prepend` is set for compilation.

Reply:
I am not sure I understand, but I assume this has something to do with expected failure of [build on JDK-11](https://travis-ci.com/github/apache/netbeans/jobs/493533632) - `java.source.base` doesn't seen the nb-javac APIs during compilation.

Reply:
Addressed in eac2f44